ktor-features/ktor-metrics-micrometer/jvm/src/io/ktor/metrics/micrometer/MicrometerMetrics.kt

```diff
@@ -88,6 +88,8 @@ public class MicrometerMetrics private constructor(
      * */
     public class Configuration {
 
+        public var baseName: String = Feature.defaultBaseName
```

Comment:
the very last thing: can you add kdoc to this and other public properties, please?